README.md

```diff
@@ -1,19 +1,38 @@
 # PolyMC Meta
+
 Scripts to generate jsons and jars that PolyMC will access.
 
-## Deployment
```

Comment:
The reason we only "support" Docker is because of some weirdness when running this on different machines. Some machines produce different output, so the Docker container is there to minimize differences in the environment.